tests/handlers/records-query.spec.ts

```diff
@@ -1608,7 +1605,7 @@ export function testRecordsQueryHandler(): void {
       });
 
       it('should return 400 if published is set to false and a datePublished range is provided', async () => {
-        const firstDayOf2021 = createDateString(new Date(2021, 1, 1));
+        const firstDayOf2021 = Time.getCurrentTimestamp();
```

Comment:
nit: rename `firstDayOf2021` to something like `startDate`

Reply:
Thanks for picking up! Fixing!